pegasus/sites.v3/code.org/views/teacher_announcements.haml

```diff
@@ -0,0 +1,16 @@
+%div.teacher-announce
+  %a.announce-heading-link{:href=>"http://teacherblog.code.org/"}
+    %div.announce-heading
+      =I18n.t(:dashboard_announcements)
+      %div.heading-view-all
+        =I18n.t(:dashboard_announce_view_all)
+        >>
+  %div.announce-content
+    %div.content-img-container
+      %img.content-img{:src=>"/images/fit-x108/kids4cropped_2x.jpg"}
+    %div.content-text
+      =I18n.t(:dashboard_announce_hoc)
+  %div.announce-learn-more
+    %a{:href=>"https://hourofcode.com"}
+      %button
+        =I18n.t(:learn_more)
```

Comment:
If it's just one thing nested in here, you can put it on the same line. I don't think we have decided either way what we want to stick to, but I think since this view is small it makes it's fine and easier to read when it's on a second line.

    %button= I18n.t(:learn_more)